wast/scanner_test.go

```diff
@@ -0,0 +1,37 @@
+package wast
+
+import (
+	"flag"
+	"fmt"
+	"testing"
+)
+
+var wastFile = flag.String("wast-file", "", "the wast file to test")
+
+func TestScanner(t *testing.T) {
```

Comment:
travis is failing because no test file is given to this `TestScanner` test.
couldn't we give it a few test files?
we have some under `exec/testdata/spec`.

start simple :)

Reply:
I think it's more suited to the situation / test case to provide a `.wast` directly in this package to modify and update as the scanner advances.

Reply:
SGTM